### DISCUSSION TITLE
Catwalks in air

### DISCUSSION THREAD

**Discussion by Badpoolinator** [BEEmod/BEE2-items#3779] [Tech Support / Q&A]
Can catwalks be suspended in the air? It's really annoying because catwalks don't really do much on the ground.

**Comment by TeamSpen210** [comment]
We can't have any custom items in mid-air, it's an editor restriction. You can place them against the wall/ceiling which resuls in them staying horizontal.

**Comment by Konclan** [comment]
You can put catwalks on lightbridges. If you turn the lightbridge off and stick it inside a wall and hide it with some blocks you can create floating scaffolding. Yes its janky but it works.
